### PR TITLE
RHMAP-6193 - Improve appearance of error message for invalid git-* pa…

### PR DESCRIPTION
**Pull request by camilamacedo86**
JIRA: https://issues.jboss.org/browse/RHMAP-6193

*Steps to Verify* 
- Create a project
- Call the build command on FHC and inform a wrong value for a git-* parameter 

*Local Test*

```
Camilas-MacBook-Pro:fh-fhc cmacedo$ ./bin/fhc.js build --app=whbc76meqqdw4pchpwwmjnot --cloud_app=whbc76ksmmcynsbvcqxxxnqv --environment=dev --destination=android --download=false --project=whbc76nzsdfnoomx23pr7qu2 --git-tag=asdfasfd
fhc ERR! Not found the git repository. Check the git parameters informed.
fhc Command executed with error
Camilas-MacBook-Pro:fh-fhc cmacedo$ 
```

### CODE REVIEW

**Comment by fh-qube** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Function has a complexity of 22 which is greater than 20 authorized.](https://github.com/feedhenry/fh-fhc/blob/64bd5f7e3d8f7c385a2d19af0b3e9902bc630935/lib/cmd/fh3/build.js#L225) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
* MINOR [Function has a complexity of 21 which is greater than 20 authorized.](https://github.com/feedhenry/fh-fhc/blob/64bd5f7e3d8f7c385a2d19af0b3e9902bc630935/lib/cmd/fh3/build.js#L227) [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AFunctionComplexity)

File: lib/cmd/fh3/build.js [hunk 1]
```diff
@@ -237,19 +256,13 @@ function doBuild(params, cb) {
       if (keys.length > 0) {
         async.map(keys, common.waitFor, function(err, results) {
           if (err) {
+            if ( err[0].error === "Error status returned from SCM: 500") {
+              return cb(i18n._("Not found the git repository. Check the git parameters informed."));
```
Comment (psturc) [inline lib/cmd/fh3/build.js]: Wdyt about this message
`Git repository not found. Please check that all provided Git parameters are valid.`

Comment (camilamacedo86) [reply lib/cmd/fh3/build.js]: 👍 Done Tks. Could you approve now?

**Comment by psturc** [comment]
@camilamacedo86 can you please squash the commits to 1? I cannot do it on Github, it's disabled for some reason.

**Pull request merged**